queries/author/author.go

```diff
@@ -0,0 +1,82 @@
+package author
+
+import (
+	"fmt"
+	"strings"
+
+	"git_applet/gitter"
+)
+
+type Result map[string]NodeMap
+
+type NodeMap struct {
+	Nodes []PullRequest `json:"nodes"`
+}
+
+type PullRequest struct {
+	Id             string                 `json:"id"`
+	Title          string                 `json:"title"`
+	Url            string                 `json:"url"`
+	BaseRefName    string                 `json:"baseRefName"`
+	HeadRefName    string                 `json:"headRefName"`
+	ReviewRequests ReviewRequest          `json:"reviewRequests"`
+	ReviewDecision string                 `json:"reviewDecision"`
+	Permalink      string                 `json:"permalink"`
+	Mergeable      string                 `json:"mergeable"`
+	Number         int                    `json:"number"`
+	State          string                 `json:"state"`
+	Review         map[string]interface{} `json:"review"`
+}
+
+type ReviewRequest struct {
+	TotalCount int `json:"totalCount"`
+}
+
+func (result NodeMap) Extract() []gitter.PullRequest {
+	all := []gitter.PullRequest{}
+
+	for _, node := range result.Nodes {
+
+		pr := gitter.PullRequest{
+			Title:          node.Title,
+			BaseRefName:    node.BaseRefName,
+			HeadRefName:    node.HeadRefName,
+			Number:         node.Number,
+			Permalink:      node.Permalink,
+			ReviewCount:    0, // TODO
```

Comment:
we should address this asap, I think there's potential here for a nice feature like: 3/4 people who reviewed approved or sth like that

Reply:
That's not a small refactor. Added it to the backlog (#35)

Reply:
yeah, this one was not for this PR :p